pandas/tests/extension/arrow/test_bool.py

```diff
@@ -55,6 +55,16 @@ def test_from_dtype(self, data):
     def test_from_sequence_from_cls(self, data):
         super().test_from_sequence_from_cls(data)
 
+    @pytest.mark.xfail(reason="GH-26469")
+    def test_series_constructor_no_data_with_index(self, data, na_value):
+        # pyarrow.lib.ArrowInvalid: only handle 1-dimensional arrays
+        super().test_series_constructor_no_data_with_index(data, na_value)
```

Comment:
Do you know why it is failing for this dtype?

Reply:
the traceback is 

```
_______________________________________________ TestConstructors.test_series_constructor_no_data_with_index ________________________________________________ 

self = <pandas.tests.extension.arrow.test_bool.TestConstructors object at 0x000001E9918B7B50>
dtype = <pandas.tests.extension.arrow.arrays.ArrowBoolDtype object at 0x000001E9918B79A0>, na_value = None

    def test_series_constructor_no_data_with_index(self, dtype, na_value):
>       result = pd.Series(index=[1, 2, 3], dtype=dtype)

pandas\tests\extension\base\constructors.py:37:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _  
pandas\core\series.py:283: in __init__
    data, index = self._init_dict(data, index, dtype)
pandas\core\series.py:372: in _init_dict
    s = create_series_with_explicit_dtype(
pandas\core\construction.py:629: in create_series_with_explicit_dtype
    return Series(
pandas\core\series.py:329: in __init__
    data = sanitize_array(data, index, dtype, copy, raise_cast_failure=True)
pandas\core\construction.py:459: in sanitize_array
    subarr = _try_cast(data, dtype, copy, raise_cast_failure)
pandas\core\construction.py:542: in _try_cast
    subarr = array_type(arr, dtype=dtype, copy=copy)
pandas\tests\extension\arrow\arrays.py:82: in _from_sequence
    return cls.from_scalars(scalars)
pandas\tests\extension\arrow\arrays.py:72: in from_scalars
    arr = pa.chunked_array([pa.array(np.asarray(values))])
pyarrow\array.pxi:265: in pyarrow.lib.array
    ???
pyarrow\array.pxi:80: in pyarrow.lib._ndarray_to_array
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _  

>   ???
E   pyarrow.lib.ArrowInvalid: only handle 1-dimensional arrays

pyarrow\error.pxi:84: ArrowInvalid
```

hmm, seems to be because using lib.is_scalar, pa.NULL is passed to sanitize_array

```
>>> import pyarrow as pa
>>>
>>> pd._libs.lib.is_scalar(pa.NULL)
False
```

Reply:
@jorisvandenbossche not sure if you want this fixed here. will raise a separate issue for this case in the meantime.

Reply:
#33899

Reply:
Yeah, that's fine, we don't really support this dtype anyway, it's only to test certain things